curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-25-5-clock.md

```diff
@@ -8,6 +8,8 @@ dashedName: build-a-25--5-clock
 
 # --description--
 
+**Note:** **React 18 has known incompatibilities with the tests for this frontend development libraries Project (see [issue](https://github.com/freeCodeCamp/freeCodeCamp/issues/45922))**
```

Comment:
```suggestion
**Note:** **React 18 has known incompatibilities with the tests for this project (see [issue](https://github.com/freeCodeCamp/freeCodeCamp/issues/45922))**
```

Capitalisation, and no need to be so verbose.

Reply:
done